conbench/db.py

```diff
@@ -5,15 +5,19 @@
 import orjson
 import psycopg2
 import sqlalchemy.exc
+import sqlalchemy.orm
 import tenacity
 from sqlalchemy import create_engine
-from sqlalchemy.orm import scoped_session, sessionmaker
 
 from .config import Config
 
 engine = None
-session_maker = sessionmaker(future=True)
-Session = scoped_session(session_maker)
+session_maker = sqlalchemy.orm.sessionmaker()
```

Comment:
What was your reasoning for dropping `future=True`? I've found that to be slightly helpful in the past, though admittedly I forget why

Reply:
Oh, here are [the docs](https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine.params.future).

>Use the 2.0 style [Engine](https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Engine) and [Connection](https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Connection) API. As of SQLAlchemy 2.0, this parameter is present for backwards compatibility only and must remain at its default value of True.

Haha. So that's why it was useful. For when we migrated to 2.0.